src/app/layout/page/overlay/overlay.component.ts

```diff
@@ -182,6 +182,14 @@ export class OverlayComponent implements OnInit, OnDestroy {
         .add(settings.exitAppKeybinding, false, VisibleFlag.Game, VisibleFlag.Overlay)
         .subscribe(() => this.app.quit())
     }
+    if (settings.restartAppUserSettingsKeybinding) {
+      this.shortcut
+        .add(settings.restartAppUserSettingsKeybinding, false, VisibleFlag.Game, VisibleFlag.Overlay)
+        .subscribe(() => {
+          this.app.relaunch()
+          this.app.quit()
```

Comment:
You don't need to call `quit` here. When calling `relaunch`, the quitting it done for you already.

Reply:
Fixed.